sdk/digitaltwins/Azure.DigitalTwins.Core/src/Customized/DigitalTwinsRestClient.cs

```diff
@@ -36,7 +36,10 @@ internal HttpMessage CreateAddRequest(string id, Stream twin, CreateDigitalTwinO
             {
                 request.Headers.Add("TraceState", digitalTwinsAddOptions.TraceState);
             }
-            request.Headers.Add("If-None-Match", "*");
+            if (digitalTwinsAddOptions?.IfNoneMatch != null)
```

Comment:
What about empty or whitespace?

Reply:
In that case the service will throw an error. 
The generated layer produces the same code so no real gain by changing the customized code.